HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -2849,52 +2850,74 @@ def self.process_neep_detailed_performance(hvac_system, mode, max_rated_fan_cfm,
-        low_odb_at_zero_capacity = calculate_odb_at_zero_cop_or_capacity(data, user_odbs, :gross_capacity, false)
-        outdoor_dry_bulbs << [low_odb_at_zero_cop, low_odb_at_zero_capacity, 55.0].max # Min cooling ODB
-        outdoor_dry_bulbs << [high_odb_at_zero_cop, high_odb_at_zero_capacity, weather_temp].min # Max cooling ODB
+        # Calculate ODB temperature at which power or capacity is zero
+        high_odb_at_zero_power = calculate_odb_at_zero_power_or_capacity(data, user_odbs, :gross_input_power, true)
+        high_odb_at_zero_capacity = calculate_odb_at_zero_power_or_capacity(data, user_odbs, :gross_capacity, true)
+        outdoor_dry_bulbs << [high_odb_at_zero_power, high_odb_at_zero_capacity, weather_temp].min # Max cooling ODB
+
+        low_odb_at_zero_power = calculate_odb_at_zero_power_or_capacity(data, user_odbs, :gross_input_power, false)
+        low_odb_at_zero_capacity = calculate_odb_at_zero_power_or_capacity(data, user_odbs, :gross_capacity, false)
+        outdoor_dry_bulbs << [low_odb_at_zero_power, low_odb_at_zero_capacity, 60.0].max # Min cooling ODB
       else
-        outdoor_dry_bulbs = []
-        # Calculate ODB temperature at which COP or capacity is zero
-        low_odb_at_zero_cop = calculate_odb_at_zero_cop_or_capacity(data, user_odbs, :gross_efficiency_cop, false)
-        low_odb_at_zero_capacity = calculate_odb_at_zero_cop_or_capacity(data, user_odbs, :gross_capacity, false)
-        high_odb_at_zero_cop = calculate_odb_at_zero_cop_or_capacity(data, user_odbs, :gross_efficiency_cop, true)
-        high_odb_at_zero_capacity = calculate_odb_at_zero_cop_or_capacity(data, user_odbs, :gross_capacity, true)
-        outdoor_dry_bulbs << [low_odb_at_zero_cop, low_odb_at_zero_capacity, compressor_lockout_temp, weather_temp].max # Min heating ODB
-        outdoor_dry_bulbs << [high_odb_at_zero_cop, high_odb_at_zero_capacity, 60.0].min # Max heating ODB
+        # Calculate ODB temperature at which power or capacity is zero
+        low_odb_at_zero_power = calculate_odb_at_zero_power_or_capacity(data, user_odbs, :gross_input_power, false)
+        low_odb_at_zero_capacity = calculate_odb_at_zero_power_or_capacity(data, user_odbs, :gross_capacity, false)
+        outdoor_dry_bulbs << [low_odb_at_zero_power, low_odb_at_zero_capacity, hp_min_temp, weather_temp].max # Min heating ODB
+
+        high_odb_at_zero_power = calculate_odb_at_zero_power_or_capacity(data, user_odbs, :gross_input_power, true)
+        high_odb_at_zero_capacity = calculate_odb_at_zero_power_or_capacity(data, user_odbs, :gross_capacity, true)
+        outdoor_dry_bulbs << [high_odb_at_zero_power, high_odb_at_zero_capacity, 60.0].min # Max heating ODB
       end
+
+      # Extrapolate net capacity and input power at those outdoor drybulb temperatures per RESNET MINHERS Addendum 82.
       capacity_description = data[0].capacity_description
       outdoor_dry_bulbs.each do |target_odb|
         next if user_odbs.include? target_odb
```

Comment:
Should really update this to check whether the target_odb is beyond the min (or max) of the user_odbs.